### PR TITLE
RandomGenerator

### PR DESCRIPTION
**Pull request by prince1111**

### CODE REVIEW

File: src/cst1201/RandomGenerator.java [hunk 1]
```diff
@@ -1,16 +1,53 @@
...
+
         //TODO: Create a Scanner object to read keyboard input.
-        
         //TODO: Ask the user a lower and upper bound.
-        
         //TODO: Ask the user for the number of random integers to generate.
-        
         //TODO: Generate random integers of the desired length and bound.
-        
         //TODO: Print the average, max, min, count, and sum of the integers.
```
Comment (khatchad) [inline src/cst1201/RandomGenerator.java]: You can remove the `TODO` once you have completed the task.

File: src/cst1201/RandomGenerator.java [hunk 2]
```diff
@@ -1,16 +1,53 @@
...
+            System.out.print("max is ");
+            System.out.println(upperbound);
+
+            System.out.print("min is ");
+            System.out.println(lowerbound);
+
+            for (int min = lowerbound; min <= upperbound; min++) {
+                System.out.print(" count:      " + min);
+
+                randomnumber = lowerbound + (int) (Math.random() * upperbound);
```
Comment (khatchad) [inline src/cst1201/RandomGenerator.java]: :-1: This is incorrect. Please use the `Random` class.

File: src/cst1201/RandomGenerator.java [hunk 3]
```diff
@@ -1,16 +1,53 @@
...
+        int randomnumber;
+        if (upperbound > lowerbound) {
+
+            System.out.print("max is ");
+            System.out.println(upperbound);
+
+            System.out.print("min is ");
+            System.out.println(lowerbound);
+
+            for (int min = lowerbound; min <= upperbound; min++) {
```
Comment (khatchad) [inline src/cst1201/RandomGenerator.java]: :-1: Lower and upper bounds are ranges, not counts.

File: src/cst1201/RandomGenerator.java (hunk 1 above)
Comment (prince1111) [inline src/cst1201/RandomGenerator.java]: I cloned it from github, so I believe  the class should have been included.

**Pull request closed without merging**